test/unit/v1.0.0/converter-v1-to-v2.test.js

```diff
@@ -1448,4 +1448,199 @@ describe('v1.0.0 to v2.0.0', function () {
             });
         });
     });
+
+    describe('query parameters', function () {
+        it('should work correctly for .convertSingle', function () {
+            transformer.convertSingle({
+                id: '0628a95f-c283-94e2-fa9f-53477775692f',
+                name: 'A world of foo!',
+                url: 'https://postman-echo.com/get?alpha&beta&gamma&delta=&epsilon=&gamma=',
+                collectionId: '03cf74df-32de-af8b-7db8-855b51b05e50',
+                queryParams: [
+                    { key: 'alpha', value: null, equals: false },
+                    { key: 'beta', value: null, equals: true },
+                    { key: 'gamma', value: null },
```

Comment:
Not sure about this though. The transformer doesn't add `''` when the value is null right?